borg/helpers.py

```diff
@@ -699,6 +705,10 @@ class Buffer:
     """
     provide a thread-local buffer
     """
+
+    class OutOfMemory(Error, OSError):
```

Comment:
That would be also preferable. In case a user ever sees that in a traceback, he should not think he has not enough RAM in the machine.